.github/workflows/build-ffmpeg.yml

```diff
@@ -32,6 +32,20 @@ jobs:
               --enable-encoder=libx264,libx265 \
               --enable-nvenc \
               --enable-vaapi
+          - os_type: linux
+            os: ubuntu-22.04
+            arch: aarch64
+            shell: bash
+            cmake_generator: Unix Makefiles
+            host: aarch64-unknown-linux-gnu
+            ffmpeg_extras: |
+              --enable-amf \
+              --enable-encoder=h264_amf,hevc_amf \
+              --enable-encoder=h264_nvenc,hevc_nvenc \
+              --enable-encoder=h264_vaapi,hevc_vaapi \
+              --enable-encoder=libx264,libx265 \
+              --enable-nvenc \
+              --enable-vaapi
```

Comment:
What’s our typical/target host system for this? Is it like a pi or a cloud instance with a GPU? Wondering if we need additional codecs

Reply:
Additional codecs are probably needed. Want to get it to build first, before messing around with other changes though.

Reply:
rpi: `h264_v4l2m2m`
jetson_nano: `h264_nvmpi`

Reply:
These can be added in a separate PR.